.github/workflows/stale.yml

```
@@ -0,0 +1,10 @@
+---
+name: stale
+
+on:
+  schedule:
```

Let's speed up testing
```suggestion
  workflow_dispatch: {}
  schedule:
```